content/posts/newsletter/2020-02-04.md

```diff
@@ -0,0 +1,262 @@
+
+Please also follow progress:
+
+* [Gajim development news January 2020](https://gajim.org/post/2020-01-27-development-news-january/)
+* [SàT progress note 2020-W01](https://www.goffi.org/b/66adNNNbFp84CWFzyvGpmK/progress-note)
+* zatrox, jbb & melvo, from the Kaidan team, explain their upcoming [streamlined registration process](https://www.kaidan.im/2020/01/08/Easy-Registration/) in the next Kaidan 0.5.0.
+
+### Libraries 
+
+Remi Corniere has released [go-xmpp v0.4.0](https://blog.process-one.net/go-xmpp-v0-4-0/).
+
+Tigase has just introduced [halcyon](https://github.com/tigase/halcyon), a Kotlin multiplatform XMPP client library.
+
+## Other 
+
+Mastodon is getting [support for XMPP URIs](https://github.com/tootsuite/mastodon/issues/9776).
+
+The EU H2020 [CPSwarm](https://www.cpswarm.eu/) project has produced as one of its main outcome the [CPSwarm Workbench](https://www.cpswarm.eu/index.php/cpswarm-workbench/), a toolchain to model, optimize, simulate and deploy a swarm of Cyber Physical Systems. One of its opensource components [released on Github](https://github.com/cpswarm), is the [Simulation and Optimization Environment](https://github.com/cpswarm/SimulationOrchestrator/wiki/Simulation-and-Optimization-Environment), which is used to evaluate the performance of a swarm solution. This environment has a distributed architecture based on XMPP, allowing to execute in parallel simulations on simulators deployed on distributed machines.
+
+## Services 
+
+Tigase has upgraded their public servers [xmpp.cloud](https://xmpp.cloud) with SASL-EXTERNAL for federation, and MUC avatars.
+
+Jonas Schäfer has described the [first update on search.jabber.network in 2020](https://sotecware.net/first-update-on-searchjabbernetwork-in-2020.html).
+
+## Extensions and specifications
+
+### Deferred 
+
+* XEP-0283 (Moved)
+* XEP-0313 (Message Archive Management)
+* XEP-0335 (JSON Containers)
+* XEP-0357 (Push Notifications)
+* XEP-0359 (Unique and Stable Stanza IDs)
+* XEP-0367 (Message Attaching)
+* XEP-0373 (OpenPGP for XMPP)
+* XEP-0384 (OMEMO Encryption)
+* XEP-0385 (Stateless Inline Media Sharing (SIMS))
+* XEP-0391 (Jingle Encrypted Transports)
+* XEP-0397 (Instant Stream Resumption)
+* XEP-0398 (User Avatar to vCard-Based Avatars Conversion)
+* XEP-0399 (Client Key Support)
+* XEP-0400 (Multi-Factor Authentication with TOTP)
+* XEP-0403 (Mediated Information eXchange (MIX): Presence Support.)
+* XEP-0404 (Mediated Information eXchange (MIX): JID Hidden Channels.)
+* XEP-0406 (Mediated Information eXchange (MIX): MIX Administration)
+* XEP-0407 (Mediated Information eXchange (MIX): Miscellaneous Capabilities)
+* XEP-0408 (Mediated Information eXchange (MIX): Co-existence with MUC)
+* XEP-0409 (IM Routing-NG)
+* XEP-0411 (Bookmarks Conversion)
+
+### Proposed 
+
+Title: Inbox
+Abstract: This specification proposes a mechanism by which clients can find a list of ongoing conversations and their state.
+URL: https://xmpp.org/extensions/inbox/inbox.html
```

Comment:
I think this won't render as expected. I'd suggest adding newlines, or better a list format like above:

* [Inbox](https://xmpp.org/extensions/inbox/inbox.html): This specification proposes a mechanism by which clients can find a list of ongoing conversations and their state.

...and the same for fulltext below!